app/lib/features/chat/widgets/convo_card.dart

```diff
@@ -296,6 +296,8 @@ class _SubtitleWidget extends ConsumerWidget {
               ],
             );
         }
+      case 'm.policy.rule.room':
```

Comment:
I thought we agreed to just not support them on legacy chat? Let's remove the changes to this file...

Reply:
Uh, I thought also chat-ng uses convo card...

Reply:
Oh, just heard from Kumar that we will replace `ChatItemWidget` in chat-ng with `ConvoCard` in legacy chat.
Will fix.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2839/commits/13e41b6d901c0e0d34c004fee9bce1a5d891ad79